packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -522,7 +522,11 @@ class DevFS {
           _scanFile(deviceUri, file);
       }
     } catch (e) {
-      // Ignore directory and error.
+      printError(
```

Comment:
This will still cause the whole package to be skipped. Would it be okay instead to put a try-catch around the file.resolveSymbolicLinksSync() call, skip the file, and emit the error message?